custom/ilsgateway/api.py

```diff
@@ -236,7 +236,7 @@ def location_sync(self, ilsgateway_location, fetch_groups=False):
             location.domain = self.domain
             location.name = ilsgateway_location.name
             if ilsgateway_location.groups:
-                location.metadata = {'groups': ilsgateway_location.groups}
+                location.metadata = {'groups': ilsgateway_location.groups[0]}
```

Comment:
confirming we think this should only contain one group now? should we change the key to be singular?